examples/combobox/combobox-datepicker.html

```diff
@@ -214,14 +214,15 @@ <h3>Date Textbox</h3>
     <ul>
       <li>Contains the date value.</li>
       <li>Opens and closes the date picker dialog box through keyboard, click and touch events.</li>
-      <li>To support people who are sighted in understanding the <kbd>down arrow</kbd> key opens the date picker dialog box, a down arrow icon appears in the <code>textbox</code> when the <code>textbox</code> receives keyboard focus and the down arrow icon is removed when the <code>textbox</code> does not have keyboard focus.</li>
+      <li>A down arrow icon in the <code>textbox</code> provides a visual affordance that the <kbd>down arrow</kbd> key opens the date picker dialog box.</li>
```

Comment:
@jongund 
These lines are going away. If you want to comment on or push changes to the accessibility features section, please do so in pr #1435